suites/squid/rbd/tier-2_rbd_group.yaml

```diff
@@ -106,13 +106,12 @@ tests:
   - test:
       desc: Cloning an image from a Group Snapshot
       module: test_rbd_groups_image_clone.py
-      name: RBD group image clone create
-      polarion-id: CEPH-83594298
+      name: RBD group image clone create and delete
+      polarion-id: CEPH-83594298 & CEPH-83594338
```

Comment:
we won't be adding multiple ids in polarion-id
if you want to combine this test here please cover those delete test steps under this test CEPH-83594298 and keep only this test.

Reply:
as it fails here 
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83594298%20&%20CEPH-83594338=

Reply:
Done